dist/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href='https://unpkg.com/boxicons@2.0.9/css/boxicons.min.css' rel='stylesheet'>
+  <title>Tv Show</title>
+<script defer src="main.js"></script></head>
+<body>
+  <header class="header">
+    <nav class="navbar">
+      <!-- <div class="logo">
+        <a href="#" class="links">NETFLEX</a>
+      </div> -->
```

Comment:
- [ ] Please remove the commented code here.

Reply:
Resolved